generators/python-v2/ast/src/__test__/__snapshots__/PythonFile.test.ts.snap

```diff
@@ -112,6 +112,17 @@ my_variable: ImportedClass.nested.attribute
 "
 `;
 
+exports[`PythonFile > Write duplicative import names 1`] = `
+"from .cars import Car as Car_1
+from .vehicles import Car as Car_2
```

Comment:
I chose a `_n` suffix, but am open to suggestions here.

Reply:
@dvargas92495 lmk if you have a preferred naming convention here.

Reply:
usually do something module dependent. eg

```python
from .cars import Car as CarsCar
from .vehicles import Car as VehiclesCar
```